index.bs

```diff
@@ -2904,7 +2904,13 @@ parameter to the {{CredentialsContainer/create()}} or {{CredentialsContainer/get
     :   <dfn>transports</dfn>
     ::  This OPTIONAL member contains a hint as to how the [=client=] might communicate with the [=managing authenticator=] of the
         [=public key credential=] the caller is referring to. The values SHOULD be members of {{AuthenticatorTransport}} but [=client platforms=] MUST ignore unknown values.
+
         The {{AuthenticatorAttestationResponse/getTransports()}} operation can provide suitable values for this member.
+        When [[#sctn-registering-a-new-credential|registering a new credential]],
+        the [=[RP]=] SHOULD store the value returned from {{AuthenticatorAttestationResponse/getTransports()}}.
+        When creating a {{PublicKeyCredentialDescriptor}} for that credential,
+        the [=[RP]=] SHOULD retrieve that stored value
+        and set it as the {{PublicKeyCredentialDescriptor/transports}} member.
```

Comment:
s/ and set it as the / and set it as the value of the /  ..?

Reply:
Thanks!